app/src/ui/branches/branch-list.tsx

```diff
@@ -53,6 +53,12 @@ interface IBranchListProps {
    */
   readonly selectedBranch: Branch | null
 
+  /** ARIA role of this element. */
+  readonly role?: string
+
+  /** ARIA labelledby of this element. */
+  readonly labelledBy?: string
```

Comment:
We have a convention to use camel case of the aria attribute. Could you please update this to use `readonly ariaLabelledBy?: string`?